profile/variables.tf

```diff
@@ -0,0 +1,45 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "instance_id" {
+  type        = string
+  description = "The ID of the SCC instance in a particular region."
+}
+
+variable "profile_name" {
+  type        = string
+  description = "The profile name. Constraints: The maximum length is `64` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "profile_description" {
+  type        = string
+  description = "The profile description. Constraints: The maximum length is `256` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "profile_type" {
+  type        = string
+  description = "The profile type, such as `custom` or `predefined`. Constraints: Allowable values are: `predefined`, `custom`."
```

Comment:
If using the profile submodule, you will always be creating a new profile right? So I dont think this should be exposed, it would always be `custom` I think